model/qti/feedback/Feedback.php

```diff
@@ -59,28 +65,27 @@ public function getBody()
     protected function getUsedAttributes()
     {
         return [
-            'oat\\taoQtiItem\\model\\qti\\attribute\\OutcomeIdentifier',
-            'oat\\taoQtiItem\\model\\qti\\attribute\\ShowHideTemplateElement'
+            OutcomeIdentifier::class,
+            ShowHideTemplateElement::class
         ];
     }
 
     /**
      * Check if the given new identifier is valid in the current state of the qti element
      *
      * @param string $newIdentifier
-     * @return booean
+     * @return bool
      * @throws InvalidArgumentException
      */
     public function isIdentifierAvailable($newIdentifier)
     {
-
         $returnValue = false;
 
         if (empty($newIdentifier) || is_null($newIdentifier)) {
             throw new InvalidArgumentException("newIdentifier must be set");
         }
 
-        if (!empty($this->identifier) && $newIdentifier == $this->identifier) {
+        if (!empty($this->identifier) && $newIdentifier === $this->identifier) {
             $returnValue = true;
         } else {
```

Comment:
Let's get rid of this ugly `else` block by introducing an early return, based on the condition above.